File: 1791-richest-customer-wealth/richest-customer-wealth.cs
public class Solution {
    public int MaximumWealth(int[][] accounts) {
        int maxwealth = 0;

         foreach (int[] customer in accounts) {
            int currwealth = 0;
            foreach (int money in customer) {
                currwealth += money;
            }
            maxwealth = Math.Max(maxwealth, currwealth);
        }
        return maxwealth;
    }
}

File: 1791-richest-customer-wealth/richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int maxwealth = 0;

        for (int i=0; i<accounts.length; i++) {
            int currwealth = 0;
            for (int j = 0; j<accounts[i].length; j++) {
                currwealth += accounts[i][j];
            }
            maxwealth = Math.max(maxwealth, currwealth);
        }
        return maxwealth;
    }
}